apps/web/src/components/PoolCandidatesTable/helpers.tsx

```diff
@@ -0,0 +1,360 @@
+import React from "react";
+import { IntlShape } from "react-intl";
+import { SortingState } from "@tanstack/react-table";
+
+import {
+  commonMessages,
+  getCandidateSuspendedFilterStatus,
+  getLanguage,
+  getPoolCandidatePriorities,
+  getPoolCandidateStatus,
+  getProvinceOrTerritory,
+} from "@gc-digital-talent/i18n";
+import { parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { Spoiler } from "@gc-digital-talent/ui";
+import {
+  OrderByRelationWithColumnAggregateFunction,
+  PoolCandidateSearchInput,
+  QueryPoolCandidatesPaginatedOrderByRelationOrderByClause,
+  QueryPoolCandidatesPaginatedOrderByUserColumn,
+} from "@gc-digital-talent/graphql";
+
+import {
+  CandidateSuspendedFilter,
+  Language,
+  PoolCandidate,
+  PoolCandidateStatus,
+  ProvinceOrTerritory,
+  SortOrder,
+  User,
+} from "~/api/generated";
+import useRoutes from "~/hooks/useRoutes";
+import { getFullNameLabel } from "~/utils/nameUtils";
+
+import cells from "../Table/cells";
+
+export const statusAccessorNew = (
```

Comment:
Does the "New" suffix mean anything, or is it just a way you avoided namespace collision while working on this ticket?

If the latter, could we remove that suffix for these functions?

Also, if this returns anything but a primitive value, it should be named "cell" since [accessors should only return primitive values](https://tanstack.com/table/v8/docs/guide/column-defs#:~:text=must%20be%20configured%20to%20extract%20primitive%20values).

---

Both comments seem to apply to all accessor named functions here outside of a few. I beleive these are being used as cells, just named incorrectly since they are not accessors.

I think renaming all of these to be `{field}Cell` instead should resolve both comments.

Reply:
> If the latter, could we remove that suffix for these functions?
Ya just left over to avoid namespace, so it will be removed.

> Also, if this returns anything but a primitive value, it should be named "cell" since [accessors should only return primitive values](https://tanstack.com/table/v8/docs/guide/column-defs#:~:text=must%20be%20configured%20to%20extract%20primitive%20values).

Okay sounds good, thanks for clearing this up.

Reply:
Fixed by [95daca8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8360/commits/95daca8101e574241aa2f72decff7b9cc7c12cdd)